openrest4j-api/src/main/java/com/openrest/olo/dispatches/PickupDispatchInfo.java

```diff
@@ -29,6 +36,14 @@ public PickupDispatchInfo clone() {
     protected PickupDispatchInfo cloneImpl() {
         return new PickupDispatchInfo(minOrderPrice, charge, delayMins, inactive,
                 ((availability != null) ? availability.clone() : null),
-                ((properties != null) ? new LinkedHashMap<>(properties) : null));
+                ((properties != null) ? new LinkedHashMap<>(properties) : null),
+                ((curbsideInfo != null) ? curbsideInfo.clone() : null),
+                withCurbsideInfo);
     }
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public Boolean withCurbsideInfo;

```

Comment:
Please = false here instead of constructor

Reply:
done